tuf/formats.py

```diff
@@ -789,20 +793,64 @@ def make_fileinfo(length, hashes, version=None, custom=None):
     will raise a 'securesystemslib.exceptions.FormatError' exception.
 
   <Returns>
-    A dictionary conformant to 'FILEINFO_SCHEMA', representing the file
+    A dictionary conformant to 'TARGETS_FILEINFO_SCHEMA', representing the file
     information of a metadata or target file.
   """
 
   fileinfo = {'length' : length, 'hashes' : hashes}
 
-  if version is not None:
-    fileinfo['version'] = version
-
   if custom is not None:
     fileinfo['custom'] = custom
 
   # Raise 'securesystemslib.exceptions.FormatError' if the check fails.
-  FILEINFO_SCHEMA.check_match(fileinfo)
+  TARGETS_FILEINFO_SCHEMA.check_match(fileinfo)
+
+  return fileinfo
+
+
+
+def make_metadata_fileinfo(version, length=None, hashes=None):
+  """
+  <Purpose>
+    Create a dictionary conformant to 'METADATA_FILEINFO_SCHEMA'.
+    This dict describes both metadata and target files.
```

Comment:
I don't think this is true? `METADATA_FILEINFO_SCHEMA` describes only metadata files